### PR TITLE
(DOCS-9356) PAR - deploy with Kubernetes

### PR DESCRIPTION
**Pull request by neko-dd**
### What does this PR do? What is the motivation?

Adds instructions for deploying a Private Action Runner with Kubernetes.

### Merge instructions

Merge queue is enabled in this repo. To have it automatically merged after it receives the required reviews, create the PR (from a branch that follows the `<yourname>/description` naming convention) and then add the following PR comment:

```
/merge
```

### CODE REVIEW

File: content/en/service_management/workflows/private_actions/use_private_actions.md [hunk 1]
```diff
@@ -76,6 +76,20 @@ From the **Private Action Runner** page in [Workflow Automation][6] or [App Buil
    You can safely ignore the error `DATADOG TRACER DIAGNOSTIC - Agent Error: connect ECONNREFUSED`.
 
 [101]: https://docs.docker.com/compose/compose-application-model/
+{{% /tab %}}
+
+{{% tab "Kubernetes" %}}
+1. Click **Kubernetes**.
+1. Confirm that you have installed `kubectl` on your machine by running `kubectl version` and verifying that there is output, then check the box on the **Private Action Runner** page.
+1. Confirm that you have installed `helm` on your machine by running `helm version` and verifying that there is output, then check the box on the **Private Action Runner** page.
+1. Confirm that you have sufficient permissions to create Kubernetes resources in your cluster by running `kubectl get pods` and verifying that the output is not an error, then check the box on the **Private Action Runner** page.<br>Further instructions appear in the app.
```
Comment (dd-gplassard) [inline content/en/service_management/workflows/private_actions/use_private_actions.md:85]: Technically running `kubectl get pods` is not sufficient to guarantee that you have all the required permissions. The exact permissions are a bit different but I don't know how much details we want to provide ? They might also change based on future updates so I'm not sure being super specific would be helpful.

File: content/en/service_management/workflows/private_actions/use_private_actions.md [hunk 2]
```diff
@@ -76,6 +76,20 @@ From the **Private Action Runner** page in [Workflow Automation][6] or [App Buil
...
+1. Follow the instructions provided in the app to:
+    1. Enroll the runner and generate the config.
+    1. Create a values.yaml file.
+    1. Add the **Private Action Runner** to your Helm repositories.
+    1. Install the Helm chart.
```
Comment (dd-gplassard) [inline content/en/service_management/workflows/private_actions/use_private_actions.md:90]: I think adding something along "run `kubectl get pods -w` and verify that the private action runner pod becomes healthy" might be useful ?

File: content/en/service_management/workflows/private_actions/use_private_actions.md (hunk 1 above)
Comment (neko-dd) [reply content/en/service_management/workflows/private_actions/use_private_actions.md:85]: Okay, I can make this less specific :)

File: content/en/service_management/workflows/private_actions/use_private_actions.md (hunk 2 above)
Comment (ksun154) [reply content/en/service_management/workflows/private_actions/use_private_actions.md:90]: Creating and testing a connection would pick up on whether the pod is responding, but it’d be helpful for the customer to double-check that the pod is fully healthy and stable before moving on. The **Ready for use** status doesn’t completely cover pod health, as it’s specific to Kubernetes, so I agree with Gabriel that adding this step would be useful.

Comment (neko-dd) [reply content/en/service_management/workflows/private_actions/use_private_actions.md:90]: Good to know, thanks. Will add now

**Review by ksun154 — APPROVED** [review]
looks great thanks!

File: content/en/service_management/workflows/private_actions/use_private_actions.md [hunk 3]
```diff
@@ -76,6 +76,21 @@ From the **Private Action Runner** page in [Workflow Automation][6] or [App Buil
    You can safely ignore the error `DATADOG TRACER DIAGNOSTIC - Agent Error: connect ECONNREFUSED`.
 
 [101]: https://docs.docker.com/compose/compose-application-model/
+{{% /tab %}}
+
+{{% tab "Kubernetes" %}}
```
Comment (neko-dd) [inline content/en/service_management/workflows/private_actions/use_private_actions.md:81]: Note for reviewer: this doc has the same stuff three times. The three copies of this new info are identical.

File: content/en/service_management/workflows/private_actions/use_private_actions.md [hunk 4]
```diff
@@ -76,6 +76,21 @@ From the **Private Action Runner** page in [Workflow Automation][6] or [App Buil
...
+1. Click **Kubernetes**.
+1. Confirm that you have installed `kubectl` on your machine by running `kubectl version` and verifying that there is output, then check the box on the **Private Action Runner** page.
+1. Confirm that you have installed `helm` on your machine by running `helm version` and verifying that there is output, then check the box on the **Private Action Runner** page.
+1. Confirm that you have sufficient permissions to create Kubernetes resources in your cluster, then check the box on the **Private Action Runner** page.<br>Further instructions appear in the app.
+1. Follow the instructions provided in the app to:
+    1. Enroll the runner and generate the config.
+    1. Create a values.yaml file.
```
Comment (aliciascott) [inline content/en/service_management/workflows/private_actions/use_private_actions.md:88]:
```suggestion
    1. Create a `values.yaml` file.
```

**Review by aliciascott — APPROVED** [review]
Hi @neko-dd small nit but approved, note I only flagged it one time but will need updated in the other 2 places :)

**Pull request merged**